wsu-people-directory.php

```diff
@@ -14,8 +14,16 @@
 	die;
 }
 
-// Flush rewrite rules on activation or deactivation.
-register_activation_hook( __FILE__, 'flush_rewrite_rules' );
+
+register_activation_hook( __FILE__, 'wsuwp_people_directory_activate' );
```

Comment:
See https://core.trac.wordpress.org/ticket/14170 and https://github.com/scribu/wp-proper-network-activation for some more backround info on activation hooks.